src/utils/sortFavorably.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Sort favored items in the beginning of the list in given order.
+ * Then sort the rest of the items alphabetically.
+ */
+export default function sortFavorably(sortable: string[], favored: string[]) {
+  favored.reverse();
```

Comment:
Would better to not mutate the array given as parameter to avoid unwanted side-effects.

```
favored.slice().reverse();
```